.gitignore

```diff
@@ -10,3 +10,5 @@ cli/lib
 test/cli/lib
 npm-debug.log
 rls*.log
+.idea
```

Comment:
#Fix: Remove this line. I recommend creating a global git ignore.

https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
fixed